cmd/get.go

```diff
@@ -84,6 +84,10 @@ func getCMDsFlags(getCmd *cobra.Command) {
 	getCmd.PersistentFlags().String("es-password", "", "ElasticSearch password to use for indexing crawl logs.")
 	getCmd.PersistentFlags().String("es-index-prefix", "zeno", "ElasticSearch index prefix to use for indexing crawl logs. Default is : `zeno`, without `-`")
 
+	// Dependencies flags
+	getCmd.PersistentFlags().Bool("no-ytdlp", false, "Disable youtube-dlp usage for video extraction.")
```

Comment:
maybe: rename the flag to `--use-ytdlp` and make the default behaviour not use `yt-dlp` as you don't want external dependencies by default

Reply:
No I think we want to use yt-dlp by default, but if it's not on the system then it won't be used, see what I mean?

Reply:
I see